### PR TITLE
Zak52 new branch

### PR DESCRIPTION
**Pull request by zak52**
I updated the documents to now have all of our documents and be able to switch between them, I updated the about page to be fully fleshed out, I made some cosmetic changes, I updated the timeline page to include the Gantt chart.

### CODE REVIEW

**Comment by amystamile** [comment]
@zak52 looks like there are some merge conflicts on a couple pages.

**Comment by zak52** [comment]
Okay I can take a look I thought I fixed them.

File: index.html [hunk 1]
```diff
@@ -52,22 +52,31 @@ <h1 class="masthead-heading mb-0">GEOSTAC</h1>
         </div>
     </header>
     <section class="page-section  text-black mb-0 bg-secondary" id="problem">
```
Comment (amystamile) [inline index.html:54]: This id needs to be renamed to something different. This is causing the problems section to not reappear when you click back on the problems tab.

File: index.html [hunk 2]
```diff
@@ -52,22 +52,31 @@ <h1 class="masthead-heading mb-0">GEOSTAC</h1>
...
+                Learn more about our project....
+              </h3>
+              <div class="divider-custom divider-secondary">
+                <div class="divider-custom-line"></div>
+                <div class="divider-custom-icon"><i class="fas fa-star"></i></div>
+                <div class="divider-custom-line"></div>
+              </div>
+              <center>
+                <iframe width="420" height="315"
+                src="https://www.youtube.com/watch?v=Tv9mCg4ejCM">
```
Comment (amystamile) [inline index.html:77]: Video is now public on YouTube. May be a different link now.

Comment (gsn9) [reply index.html:77]: this should be the link: https://youtu.be/Tv9mCg4ejCM

Comment (amystamile) [reply index.html:77]: @zak52 This is the code to use. You have to click share on YouTube then click embed to HTML.
`<iframe width="560" height="315" src="https://www.youtube.com/embed/Tv9mCg4ejCM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`

**Pull request merged**